api/src/main/java/org/openmrs/module/eptsreports/reporting/library/indicators/HivIndicators.java

```diff
@@ -26,16 +26,6 @@
 	@Autowired
 	private TxPvlsCohortQueries pvls;
 	
-	public CohortIndicator patientBelow1YearEnrolledInHIVStartedARTIndicator(CohortDefinition cohortDefinition) {
-		return newCohortIndicator("patientBelow1YearEnrolledInHIVStartedARTIndicator", EptsReportUtils.map(cohortDefinition,
-		    "onOrAfter=${startDate},onOrBefore=${endDate},location=${location},effectiveDate=${endDate}"));
-	}
-	
-	public CohortIndicator patientBetween1And9YearsEnrolledInHIVStartedARTIndicator(CohortDefinition cohortDefinition) {
-		return newCohortIndicator("patientBelow1YearEnrolledInHIVStartedARTIndicator", EptsReportUtils.map(cohortDefinition,
-		    "onOrAfter=${startDate},onOrBefore=${endDate},location=${location},effectiveDate=${endDate}"));
-	}
-	
 	public CohortIndicator patientInYearRangeEnrolledInHIVStartedARTIndicator(CohortDefinition cohortDefinition) {
```

Comment:
@ningosi can we remove all these indicators and replace with calls directly to `newCohortIndicator()`?

Reply:
Yes, I will look into those